scrabble.js

```diff
@@ -1,8 +1,173 @@
-var Scrabble = function() {};
+var Scrabble = function() {
+    // due to how the specs are written, this score table does not account for the 50 point bonus for 7-letter words...
+    this.scoreTable = [
+        [1, "A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
```

Comment:
The way this is structured could use JS objects to increase clarity. Using different data at specific indexes can oftentimes be unreliable.